com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/DecodeSH.hlsl

```diff
@@ -1,29 +1,24 @@
 #ifndef DECODE_SH
 # define DECODE_SH
 
+// TODO: We're working on irradiance instead of radiance coefficients
+//       Add safety margin 2 to avoid out-of-bounds values
+#define l1scale 2.0f // Should be: 3/(2*sqrt(3)) * 2, but rounding to 2 to issues we are observing.
```

Comment:
I'd rather we follow usual convetion for defines  (all caps, with _ between words) and don't undef if we give it a more "unique" name  (e.g. APV_L1_ENCODING_SCALE ?)